.code-samples.meilisearch.yaml

```diff
@@ -664,3 +665,23 @@ settings_guide_pagination_1: |-
       'maxTotalHits': 50
     }
   })
+async_guide_filter_by_date_1: |-
+  client.get_tasks({'afterEnqueuedAt': '2020-10-11T11:49:53.000Z'})
```

Comment:
It only works with date as strings?

Reply:
Yes, I have a problem with the standardization of dates in the SDK